### ISSUE TITLE
Compiled assets table has some problems

### ISSUE THREAD

**Issue by ingria** [JeffreyWay/laravel-mix#2721]
-   Laravel Mix Version: 6.0.5
-   Node Version: v14.15.3
-   NPM Version: 6.14.10
-   OS: Ubuntu Server 20.04 x64

### Description:

1) New table layout is less graphic than the previous "list" layout. Also it's taking too much vertical space and too little horizontal space:

![gh_shot_1](https://user-images.githubusercontent.com/2178368/103301384-5c19e600-4a12-11eb-84cb-2a4ce7d75fe4.png)

2) Files are sorted by size, not by name. It is not very convenient if there are a lot of files in the project. For example, this happens if I simply copy some directory from `node_modules` to `public`:

![gh_3](https://user-images.githubusercontent.com/2178368/103301678-e9f5d100-4a12-11eb-906d-ef41ecc73b39.png)

Compare with the old layout:

![gh_4](https://user-images.githubusercontent.com/2178368/103302209-24ac3900-4a14-11eb-91e7-6ae640fdef0b.png)

There is much more information taking up much less space (one single screen in my case, vs. **24** screens (!) in table layout).

The same project built with Mix 6:

![gh_5](https://user-images.githubusercontent.com/2178368/103302316-676e1100-4a14-11eb-95c7-9f618138ce17.png)

Also `app.css.gz` and `app.css.map` are missing (though they do exist in my `public` directory). It is now unclear whether they are being created at all.

## TL;DR

My suggestions are:

- Make the table fill all available width
- Don't limit the file name length (it's important to know the *ending* of the file path, not the *beginning*)
- Sort files by name, not by size (or maybe *sort* by size, but *group* by name)
- Fix the problem when some files are not displayed, although they should (e. g. `.map` files or files handled by webpack plugins)

**Comment by thecrypticace** [comment]
These are all really good suggestions! I'm not sure why the .map files are missing from the asset list. I'll have to look into that.
I've opened #2724 to address all but your last point about the missing assets.
